src/index.html

```diff
@@ -22,6 +22,366 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <picture class="logo-container">
+        <a
+          href="/"
+          class="logoimage"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+          />
+        </a>
+      </picture>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li>
+            <div id="apple">
+              <a
+                href="#apple"
+                class="menu-item is-active"
+              >
+                Apple
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="samsung">
+              <a
+                href="#samsung"
+                class="menu-item"
+              >
+                Samsung
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="smartphones">
+              <a
+                href="#smartphones"
+                class="menu-item"
+              >
+                Smartphones
+              </a>
+            </div>
+          </li>
+          <li>
+            <div id="laptops">
+              <a
+                href="#laptops"
+                class="menu-item"
+                data-qa="nav-hover"
```

Comment:
The `data-qa` attribute is applied to the 'Laptops & Computers' menu item. Ensure that this attribute is applied consistently across all menu items if required by the task.